docs/en/query_language/select.md

```diff
@@ -991,7 +991,7 @@ SELECT COLUMNS('a') FROM col_names
 └────┴────┘
 ```
 
-You can use multiple `COLUMNS` expressions in a query, also you can apply functions to it.
+You can use multiple `COLUMNS` expressions in a query and apply functions to them.
```

Comment:
needs clarifications:
* how actually applying functions works: columns are passed as separate arguments, what happens with additional arguments, etc
* whether an order of columns is guaranteed to be alphabetical or not